packages/components/src/components/u-i-loader/ui-loader.tsx

```diff
@@ -5,7 +5,7 @@ import Loadable from 'react-loadable';
 type TUILoader = {
     className?: string;
     classNameBlock?: string;
-} & Loadable.LoadingComponentProps;
+} & Partial<Loadable.LoadingComponentProps>;
 
 const UILoader = ({ className, classNameBlock }: TUILoader) => {
```

Comment:
why do we need different UI-loader?
cant we use UILoader from packages/components?

Reply:
I used UILoader from packages/components just edited the code to include Partial to solve the TypeScript Error